curvefs/src/mds/topology/topology.cpp

```diff
@@ -1523,11 +1523,12 @@ uint32_t TopologyImpl::GetPartitionNumberOfFs(FsIdType fsId) {
     uint32_t pNumber = 0;
     for (const auto &it : partitionMap_) {
         if (it.second.GetFsId() == fsId) {
-            pNumber++;
+            pNumber = std::max(pNumber, it.second.GetPartitionId());
```

Comment:
If the partition is deletable, shouldn't the partitioninId be discontinuous? Is it appropriate here?

Reply:
Yes, the inodeId is not be reused, so the partitionId is also can not resued if partition have been deleted.